doc/release-notes.md

```diff
@@ -57,6 +57,12 @@ Notable changes
 Example item
 ------------
 
+New settings
+------------
+
+- The `shutdownnotify` option is used to specify a command to execute synchronously
+before Bitcoin Core has begun its shutdown sequence. (#23395)
```

Comment:
As it is not clear that this pull will be merged quickly, maybe move the 5 lines to a separate file, per the instructions in https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#release-notes ?

Reply:
Done 👍🏻